server/app/core/migrator.py

```
@@ -0,0 +1,195 @@
+from sqlalchemy import text, select
+
+from . import db as _db
+from .models import Base, PluginMigration, Plugin as PluginModel
+from alembic.config import Config as AlembicConfig
+from alembic import command as alembic_command
+from pathlib import Path as _Path
+
+
+def bootstrap() -> None:
+    """Create host tables if not present."""
+    assert _db.engine is not None, "DB engine not initialized"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)